great-idea-website/css/index.css

```diff
@@ -63,4 +63,114 @@ h1, h2, h3, h4, h5 {
     margin-bottom: 15px;
 }
 
-/* Your code starts here! */
\ No newline at end of file
+/* Your code starts here! */
+
+body { 
+	padding: 0px 50px;  
+}
+
+header a {
+	display: inline-block; 
+	text-decoration: none; 
+	color: lightgray; 
+	margin: 40px; 
+	font-size: 20px; 
+}
+
+.logo {
+	display: inline-block;
+	margin-left: 220px;  
+}
+
+.main-top {
+	width: 100%; 
+	margin: auto;
+	padding-top: 50px; 
+}
+
+.main-left {
+	width: 48%; 
+	display: inline-block; 
+	font-size: 120px; 
+	text-align: center; 
+}
+
+h1 {
+	margin: 0px 0px -30px 0px; 
+
+}
+
+.main-right {
+	width: 48%; 
+	display: inline-block; 
+}
+
+.main-right img {
+	width: 60%; 
+	height: auto; 
+}
+
+button {
+	background: white; 
+	border: 2px solid black; 
+	color: black; 
+	text-decoration: none; 
+	text-align: center; 
+	font-size: 18px; 
+	padding: 10px 75px; 
+}
+
+hr {
+	width: 100%; 
+	color: black; 
+	border: 1px solid black;
+	margin-top: 75px; 
+}
+
+
+h2 {
+	font-weight: 600;
+}
+
+.wrapper {
+	font-size: 24px;
+	margin-top: 50px; 
+	
+}
+
+.middle-left {
+	width: 48%; 
+	display: inline-block; 
+}
+
+.middle-right {
+	width: 48%; 
+	display: inline-block; 
+}
+
+.middle-img {
+	width: 100%; 
+	margin: 50px 0px;  
+}
+
+.wrapper p {
+	font-size: 20px; 
+}
+
+.bottom {
+	font-size: 24px; 
+	display: inline-block; 
+	width: 30%;
+	margin: 0px 10px; 
+}
+
+footer {
+	margin-top: 50px; 
+}
+
+.copy {
+	text-align: center; 
+}
+
+
```

Comment:
try to add some more comments for some easier readability for others that look at your code and for when you revisit code much later and don't remember where you left off